### PR TITLE
Detailcard

### PR DESCRIPTION
**Pull request by JuleFloorson**

### CODE REVIEW

**Comment by nadinho** [comment]
Oh no, your deployment failed 😞

File: client/src/components/DetailCard.js [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+import Pen from '../assets/pen.svg';
+import MethodButton from './MethodButton';
+
+const Card = styled.div`
+  width: 343px;
+  height: 570px;
+  border-radius: 10px 10px 10px 10px;
+  border: 3px solid #b9b9b9;
+  color: white;
+  display: grid;
```
Comment (nadinho) [inline client/src/components/DetailCard.js:14]: Cool, grid!! 🙌

File: client/src/components/DetailCard.js [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+  grid-template-rows: 10% 8% 8% 8% auto 10%;
+  grid-template-columns: 30% auto 10% 10%;
+  align-items: center;
+`;
+
+const Heading = styled.h1`
+  text-align: left;
+  font-size: 1.25rem;
+  font-weight: 400;
+  color: #011e62;
```
Comment (nadinho) [inline client/src/components/DetailCard.js:24]: It would be good to save your colors in a file or theme, makes work easier :)

Comment (JuleFloorson) [reply client/src/components/DetailCard.js:24]: okay thanks :)

**Pull request merged**